Sender/Test_Sender.cpp

```diff
@@ -0,0 +1,17 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "test/catch.hpp"
+#include "BMS_AttributeDataSender.h"
+
+TEST_CASE("To check if file was successfully opened to read") 
+{
+ 
+  REQUIRE(BMS_Readfromdatafile()== Success); 
+}
+
+TEST_CASE("To check data is printed on the console") 
```

Comment:
This test will succeed even if we comment out the console-writing part. It is not possible to read the program behavior from the test.